app/controllers/hosts_controller.rb

```diff
@@ -491,6 +493,44 @@ def update_multiple_puppet_proxy
     redirect_back_or_to hosts_path
   end
 
+  def select_multiple_puppet_ca_proxy
+  end
+
+  def update_multiple_puppet_ca_proxy
```

Comment:
This logic is identical to that of `update_multiple_puppet_proxy`, please refactor (extract to a method, or to a concern even) to DRY.